testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -9641,6 +9641,12 @@ void testTimestampDiff(boolean coercionEnabled) {
         "2015-01-01", "DATE NOT NULL");
     f.checkScalar("date_trunc(date '2015-02-19', isoyear)",
         "2014-12-29", "DATE NOT NULL");
+    f.checkScalar("date_trunc(date '2015-02-19', decade)",
```

Comment:
Add a comment that DECADE rolls down to 0 mod 10, CENTURY and MILLENNIUM roll down to 1 mod 100 and 1 mod 1000. That behavior is surprising to some people.